src/Files/MultilingualResources/Files.ko-KR.xlf

```diff
@@ -1384,15 +1384,15 @@
         </trans-unit>
         <trans-unit id="SettingsAboutSupportUsListViewItem.AutomationProperties.Name" translate="yes" xml:space="preserve">
           <source>Support us on GitHub</source>
-          <target state="needs-review-translation">PayPal 후원하기</target>
+          <target state="translated">Github에서 도움주기</target>
           <note from="MultilingualUpdate" annotates="source" priority="2">Please verify the translation’s accuracy as the source string was updated after it was translated.</note>
         </trans-unit>
         <trans-unit id="Details" translate="yes" xml:space="preserve">
           <source>Details</source>
           <target state="translated">자세히</target>
         </trans-unit>
         <trans-unit id="No" translate="yes" xml:space="preserve">
-          <source>No</source>
+          <source>No</source>/
```

Comment:
@Johndoe0153 You have to remove the "/" at the end of the line

Reply:
Good catch!